quiche/src/dgram.rs

```diff
@@ -40,7 +42,9 @@ pub struct DatagramQueue {
 impl DatagramQueue {
     pub fn new(queue_max_len: usize) -> Self {
         DatagramQueue {
-            queue: VecDeque::with_capacity(queue_max_len),
+            queue: VecDeque::with_capacity(
```

Comment:
Seems to be a breaking change and would need a docs update? Also, will `queue_max_len` be useful in any way now?

Reply:
This only changes how much of the queue is allocated upfront. The max len limit still applies.